frontend/pages/MDMAppleSSOCallbackPage/MDMAppleSSOCallbackPage.tsx

```diff
@@ -72,7 +72,7 @@ const EnrollmentGate = ({
   return (
     <RedirectTo
       url={endpoints.MDM_APPLE_ENROLLMENT_PROFILE(
-        profileToken,
+        profileToken as string,
```

Comment:
had to add this to quiet the linter which incorrectly claimed that this could be `undefined` here.